dashboards/apigee/hybrid-cassandra-monitoring.json

```diff
@@ -0,0 +1,256 @@
+{  
+  "displayName": "Apigee Cassandra Monitoring",
+  "gridLayout": {
+    "columns": "2",
+    "widgets": [{
+      "title": "Apigee Cassandra Client READ Request Rate",
+      "xyChart": {
+        "dataSets": [{
+          "timeSeriesQuery": {
+            "timeSeriesFilter": {
+              "filter": "metric.type\u003d\"apigee.googleapis.com/cassandra/clientrequest_latency\" resource.type\u003d\"k8s_container\" metric.label.\"scope\"\u003d\"Read\" metric.label.\"unit\"\u003d\"OneMinuteRate\"",
```

Comment:
Why we use the unicode here instead of the '=' sign? It appears all the '=' signs in the PR are replaced by the unicode.